it.fabiodistasio.AntaresSQL.yml

```diff
@@ -0,0 +1,39 @@
+app-id: it.fabiodistasio.AntaresSQL
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: startantares
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
+
+modules:
+  - name: assets
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 assets/startantares /app/bin/startantares
+      - install -D it.fabiodistasio.AntaresSQL.metainfo.xml -t /app/share/metainfo/
+      - install -D assets/it.fabiodistasio.AntaresSQL.desktop -t /app/share/applications/
+      - install -D assets/it.fabiodistasio.AntaresSQL.png /app/share/icons/hicolor/256x256/apps/it.fabiodistasio.AntaresSQL.png
+    sources:
+      - type: dir
+        path: .
+
+  - name: antares
+    buildsystem: simple
+    build-commands:
+      - chmod +x Antares.AppImage
+      - ./Antares.AppImage --appimage-extract
+      - mv squashfs-root /app/bin/antares
+    sources:
+      - type: file
+        dest-filename: Antares.AppImage
+        url: https://github.com/antares-sql/antares/releases/download/v0.7.17/Antares-0.7.17-linux_x86_64.AppImage
```

Comment:
how will this work on aarch64?

Reply:
why not building from source?

Reply:
Also is the electron runtime vulnerable to the 0-day vulnerability?

Reply:
> why not building from source?

because i need to build from sources i need to run npm that needs to download modules. I have seen this method used of AppImage in other packages already published. Do you suggest a better one?


> Also is the electron runtime vulnerable to the 0-day vulnerability?

The electron update containing the patch will be in the next release.

Reply:
> > why not building from source?
> 
> because i need to build from sources i need to run npm that needs to download modules. I have seen this method used of AppImage in other packages already published. Do you suggest a better one?

Two wrongs don't make a right. 

Since you looked it up you also saw those that build from source. That's what I suggest.